data-prepper-plugins/mutate-event-processors/README.md

```diff
@@ -40,7 +40,7 @@ When run, the processor will parse the message into the following output:
 ###Configuration
 * `entries` - A list of entries to add to an event
   * `key` - The key of the new entry to be added
-  * `value` - The value of the new entry to be added
+  * `value` - The value of the new entry to be added. Strings, booleans, numbers, and null are valid to use
```

Comment:
Arrays aren’t valid? Why not?

Reply:
Ah - I didn't test for that. I was just thinking basic data types. I'll amend.